### ISSUE TITLE
False positive when using data-sap-ui-on-init with the name of a global function (v1.0.0)

### ISSUE THREAD

**Issue by codeworrior** [SAP/ui5-linter#336]
## Expected Behavior

When I use `data-sap-ui-on-init` with the name of a global function, this is still a valid and supported use case and should not result in a UI5 linter finding.

## Current Behavior

When I lint `sap.ui.core` (main branch), I get a finding for [this usage of `data-sap-ui-on-init`](https://github.com/SAP/openui5/blob/main/src/sap.ui.core/test/sap/ui/core/qunit/gherkin/fixture/testHarnessAmbiguous.html#L12).

## Context

- UI5 linter version: **`1.0.0`**

## Log Output / Stack Trace
```
12:26: Use of deprecated value 'fninit' for bootstrap parameter 'data-sap-ui-on-init
```

**Comment by codeworrior** [comment]
Thank you for the quick fix!
